### ISSUE TITLE
Improve UX on transmit button

### ISSUE THREAD

**Issue by joshrickert** [elieserdejesus/JamTaba#1335]
Hey there! I'm a relatively new user that ran into some UX hurdles. The "Transmit" button and the red/green color scheme to show the transmission state was pretty confusing to me. When I first set up the plugin I had no idea I was transmitting live while I was clicking around, and it was hard for me to figure out what the colors indicated. So I thought I'd share a suggestion.

Could the button text be updated to say "Start Transmitting" and "Stop Transmitting" depending on the state? This is how OBS handles it.

Also, thanks for making such awesome software! This is such a boon for musicians right now.

**Comment by elieserdejesus** [comment]
I agree the color logic in xmit button is inverted. And change the text to show the current status will help too.  The problemas I can see:
1 - Long texts will be a problem in VST plugin small screen. We need small texts (transmitting, not transmitting, preparing)
2 - Invert the xmit color will bug the JamTaba users.

@jonjamcam , any ideas?

**Comment by jonjamcam** [comment]
It's all in the manual :D 

![](https://raw.githubusercontent.com/wiki/elieserdejesus/JamTaba/images/Jamtaba_Manual/main%20window.jpg)

I think we should make this information more accessible. For example some apps show a 'tips' message when first installed.

As much intuitive as Jamtaba is, we can't cover everything in the main window.

**Comment by joshrickert** [comment]
@jonjamcam Yes, the diagram helps. It would help to make that more accessible for sure, because from the [plugin homepage](https://jamtaba-music-web-site.appspot.com/), the manual is pretty tough to find (4 clicks deep to its place in the Github wiki, and that's assuming I know where to look for it). I suspect a lot of musicians downloading the plugin for the first time aren't finding and reading that. I can't remember if I did.

As far as the UX on the button itself, if the button is size-constrained, perhaps the transmission state could be communicated through an icon (airwaves?) or separate status text? Or the button could just say "Start/Stop" with a separate "Transmission" label. I'm just brainstorming here.
